components/callbacks.py

```diff
@@ -428,3 +427,22 @@ async def say_potato_command(update: Update, context: ContextTypes.DEFAULT_TYPE)
         ),
         name=f"POTATO {user.id}",
     )
+
+
+async def buy(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
+    """Tells people to not do job offers in our group"""
+    message = cast(Message, update.effective_message)
+    who_banned = cast(User, message.from_user)
+    chat = cast(Chat, update.effective_chat)
+
+    await message.delete()
```

Comment:
I think a wrote a `try_to_delete` wrapper somewhere that just ignores TG exceptions. Might be good to use here as we want the rest to be executed even if deleting this message fails

Reply:
should I do the same for potato_command, where it also deletes the triggering message?

Reply:
sure, why not :)